SpatialGDK/Source/SpatialGDK/Private/EngineClasses/Components/ActorInterestComponent.cpp

```diff
@@ -5,36 +5,29 @@
 #include "Schema/Interest.h"
 #include "Interop/SpatialClassInfoManager.h"
 
-void UActorInterestComponent::CreateQueries(const USpatialClassInfoManager& ClassInfoManager, const SpatialGDK::QueryConstraint& AdditionalConstraints, TArray<SpatialGDK::Query>& OutQueries) const
+void UActorInterestComponent::PopulateFrequencyToConstraintsMap(const USpatialClassInfoManager& ClassInfoManager, TMap<float, TArray<SpatialGDK::QueryConstraint>>& OutFrequencyToQueryConstraints) const
```

Comment:
Usually for complex containers it's worth adding a `using`

Reply:
fixed